e2e/infra/cluster_operations.go

```
@@ -35,17 +38,35 @@ func LoginAsTestPowerUser() {
 }
 
 func ClientVersion() int {
+	if version, found := os.LookupEnv("IKE_CLUSTER_VERSION"); found {
+		result, _ := strconv.ParseInt(version, 0, 0)
```

Atoi?

---

and please don't ignore the error :)

---

Sloppy me. Fixed with 4bb2a48